From 8c962fdb73e64deec1a2b45436d383be51585bbe Mon Sep 17 00:00:00 2001
From: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Date: Tue, 3 May 2022 18:20:36 +0200
Subject: [PATCH 0241/1141] dt-bindings: remoteproc: Fix error and warning

Fix error and warning reported by dt_binding_check.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>
Change-Id: Ib888badf63d3d219b5fa5cf537daa4ea1e3ef8ce
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/253639
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 .../devicetree/bindings/remoteproc/st,stm32-rproc.yaml       | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml b/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
index 3450afec60b0..484ca03e53e7 100644
--- a/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
+++ b/Documentation/devicetree/bindings/remoteproc/st,stm32-rproc.yaml
@@ -32,7 +32,6 @@ properties:
     items:
       - const: mcu_rst
       - const: hold_boot
-    minItems: 1
 
   st,syscfg-holdboot:
     description: remote processor reset hold boot
@@ -146,7 +145,9 @@ examples:
       reg = <0x10000000 0x40000>,
             <0x30000000 0x40000>,
             <0x38000000 0x10000>;
-      resets = <&rcc MCU_R>;
+      resets = <&scmi_reset RST_SCMI_MCU>,
+               <&scmi_reset RST_SCMI_MCU_HOLD_BOOT>;
+      reset-names = "mcu_rst", "hold_boot";
       st,syscfg-holdboot = <&rcc 0x10C 0x1>;
       st,syscfg-rsc-tbl = <&tamp 0x144 0xFFFFFFFF>;
       st,syscfg-m4-state = <&tamp 0x148 0xFFFFFFFF>;
-- 
2.39.2

